crates/proto/src/rr/domain/name.rs

```diff
@@ -1450,6 +1450,7 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::same_item_push)]
```

Comment:
Also see aforementioned link for an alternative suggestion.

Reply:
yup, I have this

`bytes.extend((0..=n).into_iter().map(|_| n));`